src/components/pages/text-curation/edit/table/form/index.tsx

```diff
@@ -48,16 +51,41 @@ export default function EditRowForm({ row }) {
       ...values,
       curatedSName: values.curatedSName || ""
     });
+    const btn: any = document.querySelector(`[data-testid=expander-button-${row.id}]`);
+    btn.click();
   };
 
   return (
     <Box borderLeft="1px" borderColor="gray.200" h="full">
       <FormProvider {...hForm}>
         <form onSubmit={hForm.handleSubmit(handleOnSubmit)}>
-          <ScientificNameEdit row={row} />
-          <LocationEdit row={row} />
-          <DateEdit row={row} />
-          <EditStatus />
+          <Box borderColor="gray.300" mb={4} borderWidth="thin">
```

Comment:
Pls try to write a generic component as the below line are DRY code

Reply:
Done.